src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -3,6 +3,20 @@
 
 public class Task3 {
     public static String comboString(String s1, String s2) {
-        return "";
+        if (s1 == null){
+            s1 = String.valueOf(s1);
+        }
+
+        if (s1.length() > s2.length())
+        {
+            StringBuffer newstring = new StringBuffer();
+            newstring.append(s2).append(s1).append(s2);
+            return newstring.toString();
+        }
+        else {
+            StringBuffer newstring2 = new StringBuffer();
+            newstring2.append(s1).append(s2).append(s1);
+            return newstring2.toString();
+        }
```

Comment:
There is no need for `StringBuffer` in current task.
Also, `StringBuffer` use `synchronized` methods, so your code will be additionally slowed by synchronization.